src/nvim/api/vim.c

```diff
@@ -2389,3 +2389,144 @@ Dictionary nvim_complete_set(Integer index, Dict(complete_set) *opts, Arena *are
   }
   return rv;
 }
+
+static void redraw_status(win_T *wp, Dict(redraw) *opts, bool *flush)
+{
+  if (opts->statuscolumn && *wp->w_p_stc != NUL) {
+    wp->w_nrwidth_line_count = 0;
+    changed_window_setting(wp);
+  }
+  win_grid_alloc(wp);
+
+  // Flush later in case winbar was just hidden or shown for the first time, or
+  // statuscolumn is being drawn.
+  if (wp->w_lines_valid == 0) {
+    *flush = true;
+  }
+
+  // Mark for redraw in case flush will happen, otherwise redraw now.
+  if (*flush && (opts->statusline || opts->winbar)) {
+    wp->w_redr_status = true;
+  } else if (opts->statusline || opts->winbar) {
+    win_check_ns_hl(wp);
+    if (opts->winbar) {
+      win_redr_winbar(wp);
+    }
+    if (opts->statusline) {
+      win_redr_status(wp);
+    }
+    win_check_ns_hl(NULL);
+  }
+}
+
+/// Instruct Nvim to redraw various components.
+///
```

Comment:
```suggestion
///
/// @see |:redraw|
///
```

Let's also mention `nvim_redraw()` from the docs for :redraw/:redrawstatus/etc.

Reply:
Sure. Could add a line like this to each command

    See |nvim_redraw()| for a more granular alternative.

Or add some kind of header above the bunch of them as those commands are grouped.

Reply:
> Could add a line like this to each command

Just `See also |nvim_redraw()|.` on each command is the usual pattern. Headers are less effective for "discovery".